nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -52,6 +52,10 @@ def list_available_models(cls) -> Optional[PretrainedModelInfo]:
         return result
 
     def __init__(self, cfg: DictConfig, trainer=None):
+        # Convert to Hydra 1.0 compatible DictConfig
+        cfg = model_utils.setup_model_config(cfg)
```

Comment:
(1) are these two always called in sequence one after another?
(2) should every child of ModelPT do these calls now?

Reply:
(1)

The first call is required if the Model needs to change something's before calling super().init(). (aka inside BPE models that change decoder tokens on the fly). Most subclasses of ModelPT won't need to explicitly call this as super () will do it for them. 

The second fall needs to be done after the first one (and is done by ModelPT as well). But for models that need this before actually falling super() it needs to be done manually. 

(2) Very special cases are required to call these two manually. Most models can do without them (aka rest or entire CTC/RNNT for char based, NLP, NMT and TTS)